fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -129,6 +129,7 @@ private static List<StatementBase> tryParseWithStarRocksDialect(String sql, Sess
     }
 
     private static List<StatementBase> parseWithStarRocksDialect(String sql, SessionVariable sessionVariable) {
+        LOG.info("origin sql is {}", sql);
```

Comment:
should be removed

Reply:
> should be removed

This is just a draft PR and I submit this draft pr just to facilitate some testing.  I will split it into several PRs and resubmit, I think you can review later.